content/contribute/code.md

```diff
@@ -0,0 +1,124 @@
+---
+How to contribute to Wazo Platform code base
```

Comment:
`to Wazo Platform code base` => `to the Wazo Platform code base`

Reply:
I prefer the former wording.